common/p_spec.cpp

```diff
@@ -2246,13 +2426,14 @@ DScroller::DScroller (EScrollType type, fixed_t dx, fixed_t dy,
 	if ((m_Control = control) != -1)
 	{
 		sector_t *sector = &sectors[control];
-		fixed_t height = P_FloorHeight(sector->soundorg[0], sector->soundorg[1], sector);
+		fixed_t height = sector->ceilingheight;
```

Comment:
See above - why get rid of `P_FloorHeight` - aside from the fact that it's floor height and not ceiling height?